pp/wifipiano2.py

```diff
@@ -82,14 +82,14 @@ def getPP(self):
 				hitWindow300 *= 1.4
 
 			# Fiddles with DT and HT to make them match hitWindow300's ingame.
-			if scoreMods & mods.DOUBLETIME or mods.NIGHTCORE != 0:
+			if scoreMods & mods.DOUBLETIME | mods.NIGHTCORE != 0:
```

Comment:
```suggestion
			if scoreMods & (mods.DOUBLETIME | mods.NIGHTCORE) != 0:
```
pake dalam kurung buat cek prioritas, kalo gak sama aja soalnya.
also `!= 0` nya optional, kalo gw sih gak pake.
(ini berlaku buat sisanya juga)

Reply:
`!=0` coba gw hapus dah

Reply:
semuanya itu kan? atau ada yg bbrp harus pakai `! = 0` ?

Reply:
itu optional karena dalam python, buat `int` type itu bool valuenya `0` atau `bukan 0`, makanya kalo `if 0` itu sama aja kaya `if False`, dan sebaliknya `if True` itu bisa `if 1` atau `if 4294967296` sekalipun -w-